.github/workflows/ci.yml

```diff
@@ -36,7 +33,7 @@ jobs:
           - 5432:5432
 
       mariadb:
-        image: mariadb:10.5
+        image: mariadb:10
```

Comment:
I don't believe this change was necessary.

Reply:
https://github.com/moodlehq/moodle-plugin-ci/blob/master/gha.dist.yml#L20

the 10 tag points to the latest version in the 10.x series so 10.11.y
Whereas 10.5 limits to 10.5.y
https://hub.docker.com/_/mariadb/tags?page=2

it was pinned at some point but issue  was resolved so pin aught to be removed

Reply:
https://github.com/moodlehq/moodle-plugin-ci/commit/4907b25c05d82bc616f3f907cc76d9b9339d3017

Reply:
Okay. But in the spirit of once issue per change, this does not need to be here. If we need to update the ci file, we will need another issue/pull request.

Reply:
@opitz